version: "3"

services:
  uptime_kuma:
    image: "louislam/uptime-kuma:1"
    ports:
      - "3003:3001"
    volumes:
      - uptime_kuma_data:/app/data:rw
      - "/var/run/docker.sock:/var/run/docker.sock:ro"
    environment:
      - TZ=America/Chicago
      - PGID=999
      - PUID=0
    restart: unless-stopped
    # labels:
    #   - "traefik.enable=false"
    #   - "traefik.http.routers.uptime_kuma.rule=Host(`uptime_kuma.ansible_nas_domain`)"
    #   - "traefik.http.routers.uptime_kuma.tls.certresolver=letsencrypt"
    #   - "traefik.http.routers.uptime_kuma.tls.domains[0].main=ansible_nas_domain"
    #   - "traefik.http.routers.uptime_kuma.tls.domains[0].sans=*.ansible_nas_domain"
    #   - "traefik.http.services.uptime_kuma.loadbalancer.server.port=3001"

volumes:
  uptime_kuma_data:
